nflapi/helpers.py

```diff
@@ -144,10 +146,53 @@ def by_ids(self, ids: List[str], select_fun: Callable[[shield.Team], None] = Non
         return teams.viewer.teams_by_ids
 
 
+class RosterHelper(Helper):
+    def lookup(self, abbreviation, select_fun: Callable[[shield.Team], None] = None):
+        def add_abbr_and_property_id(team):
+            if select_fun:
+                select_fun(team)
+            team.abbreviation()
+            team.franchise.property.id()
+
+        all_teams = self.nfl.team.get_all(select_fun=add_abbr_and_property_id)
+        the_team = next(filter(lambda t: t.abbreviation == abbreviation, all_teams), None)
+        if not the_team:
+            return None
+
+        return self.by_id(the_team.franchise.property.id, select_fun)
+
+    def by_id(self, id: str, select_fun: Callable[[shield.Team], None] = None):
+        # id param is team.franchise.property.id
+        op = Operation(shield.Viewer)
+        roster = op.viewer.clubs.current_club_roster(property_id=id)
+        apply_selector(roster, shield.CurrentClubRoster, select_fun)
+        roster = self.query(op)
+        return roster.viewer.clubs.current_club_roster
+
+    def by_team_id(self, team_id: str, select_fun: Callable[[shield.Team], None] = None):
+        def add_property_id(team):
+            if select_fun:
+                select_fun(team)
+            team.franchise.property.id()
+
+        the_team = self.nfl.team.by_ids(ids=[team_id], select_fun=add_property_id)
+        if not len(the_team):
+            return None
+        else:
+            the_team = the_team[0]
+
+        return self.by_id(the_team.franchise.property.id, select_fun)
+
+
+class InjuryHelper(Helper):
```

Comment:
This is going to make someone sad - are you planning to fill it in later? If so I don't mind merging it as a stub but if it was an accident it should probably be removed.

Reply:
Planning to add later, but did not intend to include. Removed!